docs/streams/developer-guide/dsl-api.html

```diff
@@ -3286,13 +3306,33 @@ <h5><a class="toc-backref" href="#id34">KTable-KTable Foreign-Key
-                            within the window size.  Thus, sliding windows are not aligned to the epoch, but to the data record timestamps.  In
-                            contrast to hopping and tumbling windows, the lower and upper window time interval bounds of sliding windows are
-                            <em>both inclusive</em>.</p>
+                            within the window size. As a sliding window moves along the time axis, records may fall into multiple snapshots of
+                            the sliding window, but each unique combination of records appears only in one sliding window snapshot.</p>
+                        <p>The following code defines a sliding window with a time difference of 10 minutes and a grace period of 30 minutes:</p>
+                        <div class="highlight-java"><div class="highlight"><pre><span></span><span class="kn">import</span> <span class="nn">org.apache.kafka.streams.kstream.SlidingWindows</span><span class="o">;</span>
+
+<span class="c1">// A sliding time window with a time difference of 10 minutes and grace period of 30 minutes</span>
+<span class="kt">Duration</span> <span class="n">timeDifferenceMs</span> <span class="o">=</span> <span class="n">Duration</span><span class="o">.</span><span class="na">ofMinutes</span><span class="o">(</span><span class="mi">10</span><span class="o">);</span>
+<span class="kt">Duration</span> <span class="n">gracePeriodMs</span> <span class="o">=</span> <span class="n">Duration</span><span class="o">.</span><span class="na">ofMinutes</span><span class="o">(</span><span class="mi">30</span><span class="o">);</span>
+<span class="n">SlidingWindows</span><span class="o">.</span><span class="na">withTimeDifferenceAndGrace</span><span class="o">(</span><span class="n">timeDifferenceMs</span><span class="o">,</span><span class="n">gracePeriodMs</span><span class="o">);</span>
+</pre></div>
+                         <div class="admonition note">
+                             <p><b>Note</b></p>
+                             <p>Sliding windows <em>require</em> that you set a grace period, as shown above. For time windows and session windows,
+                                 setting the grace period is optional and defaults to 24 hours.</p>
+                         </div>
+                         <div class="figure align-center" id="id35">
+                             <img class="centered" src="/{{version}}/images/streams-sliding-windows.png">
+                             <p class="caption"><span class="caption-text">This diagram shows windowing a stream of data records with sliding windows. The overlap of
+                                 the sliding window snapshots varies depending on the record times. In this diagram, the time numbers represent miliseconds. For example,
+                                 t=5 means “at the five milisecond mark”.</span></p>
```

Comment:
I rendered the site locally and there's something weird going on here. Seems like you can't use literal `"`. Look at what the other captions do, seems like it might be `&#8220;`